src/agent/border_agent.cpp

```diff
@@ -280,7 +280,7 @@ void BorderAgent::SetThreadVersion(uint16_t aThreadVersion)
 #endif
 }
 
-void BorderAgent::HandlePSKc(void *aContext, int aEvent, va_list aArguments)
+void BorderAgent::HandlePSKc(void *aContext, OTBR_UNUSED int aEvent, va_list aArguments)
```

Comment:
nit: if we start using `OTBR_UNUSED`, how about putting after parameter declaration? I think it formats better when parameters are split across multiple lines.

Example from other projects: https://github.com/apple/HomeKitADK/search?q=UNUSED&unscoped_q=HAP_UNUSED

Reply:
Ok, I will follow their style.